File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitExpLogZ.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitExpLogZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("e^(log z)")
                .computeFn(w -> w.region1mapped(z -> z.log().exp()))
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitExpLogZ.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitExpLogZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("e^(log z)")
        .computeFn { w -> w.region1mapped { z -> z.log.exp } }
        .render()
}